### PR TITLE
Merging GitBook Branch! 

### PR DESCRIPTION
**Pull request by AlannaBurke**
This PR merges all changes that have been made to documentation since forking off the repo to work on GitBook. The only changes should be to documentation. Some structure was changed inside of the `docs/` directory but that should be sorted here.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -1,24 +0,0 @@
----
```
Comment (tobybellwood) [inline .github/ISSUE_TEMPLATE/bug_report.md]: Do we need to remove this?

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -1,26 +0,0 @@
- <!--
```
Comment (tobybellwood) [inline .github/PULL_REQUEST_TEMPLATE.md]: likewise, do we need to remove this file?

File: .gitmodules [hunk 3]
```diff
@@ -0,0 +1,3 @@
+[submodule "documentation"]
```
Comment (tobybellwood) [inline .gitmodules]: what is this one?

**Review by tobybellwood — COMMENTED** [review]
some initial q's

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (AlannaBurke) [reply .github/ISSUE_TEMPLATE/bug_report.md]: No idea, I think GitBook might have run amok here, I'll clean it up.

File: .gitmodules (hunk 3 above)
Comment (AlannaBurke) [reply .gitmodules]: Oh, that was my ill-fated attempt to get this working with sub-modules, thanks for catching this, I thought I'd deleted it.

**Review by tobybellwood — APPROVED** [review]
Let's do this!!!

**Pull request merged**